com.sejda.Desktop.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.sejda.Desktop</id>
+  <name>Sejda PDF Desktop</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Sejda BV</developer_name>
+  <summary>Productive PDF software that you'll love to use</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.sejda.Desktop.desktop</launchable>
+  <url type="homepage">https://www.sejda.com/desktop</url>
+  <description>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Sejda BV.</p>
+    <p>Sejda PDF Desktop is a PDF editor, and it is free to use with daily limits.</p>
+    <p>We help with your PDF tasks:</p>
+    <ul>
+      <li>Merge files into one PDF</li>
+      <li>Split PDF files</li>
+      <li>Convert PDF files to Office and image formats</li>
+      <li>Compress PDF</li>
+      <li>Edit PDF files</li>
+      <li>Apply PDF passwords and restrictions</li>
+      <li>Watermark PDF</li>
+      <li>Grayscale PDF</li>
+      <li>Crop PDF</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/libera-free/flathub/com.sejda.Desktop/screenshot.png</image>
```

Comment:
this should NEVER link to a branch. Because a branch lives, content dies with it.

Reply:
Fixed